lib/OnyxCache.js

```diff
@@ -191,6 +191,16 @@ class OnyxCache {
     setRecentKeysLimit(limit) {
         this.maxRecentKeysSize = limit;
     }
+
+    /**
+     * @param {String} key
+     * @param {*} value
+     * @returns {Boolean}
+     */
+    hasValueChanged(key, value) {
+        this.addToAccessedKeys(key);
```

Comment:
Kind of odd to call this here, no? Maybe it would be better to add it [here](https://github.com/Expensify/react-native-onyx/pull/191/files#diff-146c57cda221f8f5ee5c808eee0c6692b02c4ac2b8818d0104896a04be78e78aR689)?

Reply:
yep that will work thanks!